Header header
string hostname
uint32 total_num_of_messages
float64 total_bytes_per_second
float32 total_avg_msgs_per_sec
float32 total_avg_bytes_per_msg
uint64 total_bytes
SingleTopicStats[] topics